libs/unity/library/Runtime/Scripts/Media/MediaLine.cs

```diff
@@ -50,12 +52,23 @@ public class MediaLine : ISerializationCallbackReceiver
         /// If <see cref="Transceiver"/> is valid, that is a first session negotiation has already been completed,
         /// then changing this value raises a <see xref="WebRTC.PeerConnection.RenegotiationNeeded"/> event on the
         /// peer connection of <see cref="Transceiver"/>.
-        /// </summary>
+        ///
+        /// Must be changed on the main thread.
```

Comment:
Avoid confusion with _e.g._ the UWP main UI thread:
```suggestion
        /// Must be changed on the main Unity app thread.
```